### PR TITLE
Adds FARfilter support

### PR DESCRIPTION
**Pull request by osstony**
Addresses #41.

Please take a look and let me know if you'd like me to change anything. The functionality is present but I'm not 100% sure it's coded in the best way.

Perhaps line 289 shouldn't be hardcoded and an optional kwarg should be added.

### CODE REVIEW

File: Source/UnrealEnginePython/Private/UEPyEditor.cpp [hunk 1]
```diff
@@ -264,6 +264,44 @@ PyObject *py_unreal_engine_get_assets(PyObject * self, PyObject * args) {
...
 
+PyObject *py_unreal_engine_get_assets_by_filter(PyObject * self, PyObject * args) {
+
+	PyObject *pyfilter;
+
+	if (!PyArg_ParseTuple(args, "|O:get_assets_by_class", &pyfilter)) {
+		return NULL;
+	}
+
+	if (!pyfilter || pyfilter == Py_None) {
```
Comment (rdeioris) [inline Source/UnrealEnginePython/Private/UEPyEditor.cpp]: not needed, just change '|O:get_assets_by_class' to 'O:get_assets_by_filter'

File: Source/UnrealEnginePython/Private/UEPyFARFilter.cpp [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+}
+
+static PyMethodDef ue_PyFARFilter_methods[] = {
+	{ "append", (PyCFunction)py_ue_farfilter_append, METH_VARARGS, "" },
+	{ "clear", (PyCFunction)py_ue_farfilter_clear, METH_VARARGS, "" },
+	{ "is_empty", (PyCFunction)py_ue_farfilter_is_empty, METH_VARARGS, "" },
+	{ NULL }  /* Sentinel */
+};
+
+static PyObject *py_ue_farfilter_get_iooda(ue_PyFARFilter *self, void *closure) {
```
Comment (rdeioris) [inline Source/UnrealEnginePython/Private/UEPyFARFilter.cpp]: better to use explicit names instead of acronysms,, it is boring but more 'pythonic' :)

Comment (osstony) [inline Source/UnrealEnginePython/Private/UEPyFARFilter.cpp]: Will do. The name was just going to be really really long and I wasn't sure which was worse :).

**Comment by osstony** [comment]
The PR has been updated with your notes

**Comment by rdeioris** [comment]
Merged, thanks

**Pull request merged**